src/lib/components/Timeline/TimelinePoint.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC, PropsWithChildren, ComponentProps } from 'react';
+import classNames from 'classnames';
+import { useTimelineContext } from './TimelineContext';
+
+export type TimelnePointProps = PropsWithChildren<{ icon?: FC<ComponentProps<'svg'>> }>;
+
+export const TimelinePoint: FC<TimelnePointProps> = ({ children, icon: Icon, ...props }) => {
+  const { horizontal } = useTimelineContext();
+  return (
+    <div data-testid="timeline-point" className={classNames({ 'flex items-center': horizontal })} {...props}>
+      {children}
+      {Icon ? (
+        <span className="absolute -left-3 flex h-6 w-6 items-center justify-center rounded-full bg-blue-200 ring-8 ring-white dark:bg-blue-900 dark:ring-gray-900">
```

Comment:
This needs to either not be visible to screen readers, or pass in an accessible name

Reply:
You might have to give me hand on this one. I'm not sure what the best way to do this is. :(

Reply:
I would probably just add the sr-only class (hide this icon for screen readers) since each icon will be accompanied by relevant text already.

Reply:
> I would probably just add the sr-only class (hide this icon for screen readers) since each icon will be accompanied by relevant text already.

Okay so do I just add the sr-only class and make it a conditional based on whether or not there is an Icon. This would also replace the ternary statement right?